src/tests/multihost/alltests/test_ldap_password_policy.py

```diff
@@ -0,0 +1,277 @@
+""" Automation of password Policy suite
+:requirement: ldap_id_ldap_auth
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:status: approved
+"""
+import time
+import pytest
+import ldap
+import os
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from constants import ds_instance_name, ds_rootpw
+
+
+def find_logs(multihost, log_name, string_name):
+    """This function will find strings in a log file
+    log_name: Absolute path of log where the search will happen.
+    string_name: String to search in the log file.
+    """
+    log_str = multihost.client[0].get_file_contents(log_name).decode('utf-8')
+    assert string_name in log_str
+
+
+@pytest.fixture(scope='function')
+def common_sssd_setup(multihost):
+    """
+    This is common sssd setup used in this test suit.
+    """
+    tools = sssdTools(multihost.client[0])
+    tools.sssd_conf("nss", {'filter_groups': 'root',
+                            'filter_users': 'root',
+                            'debug_level': '9'}, action='update')
+    ldap_params = {'use_fully_qualified_names': False}
+    tools.sssd_conf(f'domain/{ds_instance_name}', ldap_params)
+    tools.clear_sssd_cache()
+
+
+def ldap_modify_ds(multihost, ldap_mode, dn_dn, element, value):
+    """This funciton will do modification on master elements
+    ldap_mode: Mode of ldap operation.
+    dn_dn: Dn to operate.
+    element: Attribute of Dn which needs to change
+    value: New Attribute value.
+    """
+    ldap_uri = f'ldap://{multihost.master[0].sys_hostname}'
+    ds_rootdn = 'cn=Directory Manager'
+    ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
```

Comment:
ds_rootdn is used only once in fixture, you could directly use it Or add it in constants.py and import it from there.